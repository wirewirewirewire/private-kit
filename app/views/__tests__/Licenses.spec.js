import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Licenses from '../Licenses';

it('renders correctly', () => {
  const tree = renderer
    .create(<Licenses />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
