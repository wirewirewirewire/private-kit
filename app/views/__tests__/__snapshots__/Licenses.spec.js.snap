// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#FFF",
      "color": "#000",
      "flex": 1,
      "flexDirection": "column",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "borderBottomColor": "rgba(189, 195, 199,0.6)",
        "borderBottomWidth": 1,
        "flexDirection": "row",
        "height": 60,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "height": 60,
          "opacity": 1,
          "paddingLeft": 20,
          "paddingTop": 21,
          "width": 60,
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../app/assets/images/backArrow.png",
          }
        }
        style={
          Object {
            "height": 18,
            "width": 18.48,
          }
        }
      />
    </View>
    <Text
      style={
        Object {
          "fontFamily": "OpenSans-Bold",
          "fontSize": 24,
        }
      }
    >
      Licenses
    </Text>
  </View>
  <View
    style={
      Object {
        "alignSelf": "center",
        "flex": 1,
        "flexDirection": "column",
        "padding": 20,
        "textAlignVertical": "top",
        "width": "96%",
      }
    }
  >
    <Text
      style={
        Object {
          "fontFamily": "OpenSans-Bold",
          "fontSize": 24,
        }
      }
    >
      Private Kit
    </Text>
    <View
      style={
        Object {
          "backgroundColor": "#FFF",
          "color": "#000",
          "flex": 1,
          "flexDirection": "row",
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 20,
            "fontWeight": "800",
          }
        }
      >
        Version: 
      </Text>
      <Text
        style={
          Object {
            "fontSize": 20,
            "fontWeight": "200",
          }
        }
      >
        0.5.9
      </Text>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#FFF",
          "color": "#000",
          "flex": 1,
          "flexDirection": "row",
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 20,
            "fontWeight": "800",
          }
        }
      >
        OS: 
      </Text>
      <Text
        style={
          Object {
            "fontSize": 20,
            "fontWeight": "200",
          }
        }
      >
        ios vundefined
      </Text>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#FFF",
          "color": "#000",
          "flex": 1,
          "flexDirection": "row",
        }
      }
    >
      <Text
        style={
          Object {
            "fontSize": 20,
            "fontWeight": "800",
          }
        }
      >
        Screen Resolution: 
      </Text>
      <Text
        style={
          Object {
            "fontSize": 20,
            "fontWeight": "200",
          }
        }
      >
         
        750 x 1334
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 4,
        "paddingLeft": 20,
        "paddingRight": 15,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "flex": 1,
            "overflow": "hidden",
          },
          undefined,
        ]
      }
    >
      <RNCWebView
        cacheEnabled={true}
        javaScriptEnabled={true}
        messagingEnabled={false}
        onContentProcessDidTerminate={[Function]}
        onHttpError={[Function]}
        onLoadingError={[Function]}
        onLoadingFinish={[Function]}
        onLoadingProgress={[Function]}
        onLoadingStart={[Function]}
        onMessage={[Function]}
        onShouldStartLoadWithRequest={[Function]}
        source={
          Object {
            "html": "<html><style>  html, body { font-size: 40px; margin: 0; padding: 0; } </style><body><B>Private Kit</B><P>MIT License<br/><br/>Copyright (c) 2020 TripleBlind, Inc<br/><br/>Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the \\"Software\\"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:<br/><br/>The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.<br/><br/>THE SOFTWARE IS PROVIDED \\"AS IS\\", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.<hr/></body></html>",
          }
        }
        style={
          Array [
            Object {
              "flex": 1,
              "overflow": "hidden",
            },
            Object {
              "backgroundColor": "#ffffff",
            },
            Object {
              "marginTop": 15,
            },
          ]
        }
        useSharedProcessPool={true}
      />
    </View>
  </View>
</RCTSafeAreaView>
`;
